bash_completion

```diff
@@ -10,7 +10,7 @@ __nvm_generate_completion()
 {
   declare current_word
   current_word="${COMP_WORDS[COMP_CWORD]}"
-  COMPREPLY=($(compgen -W "$1" -- "$current_word"))
+  COMPREPLY=("$(compgen -W "$1" -- "$current_word")")
```

Comment:
is not the intention to split the output here?

Reply:
My fault.